docker.gradle

```diff
@@ -6,6 +6,9 @@ DOCKER_REPO_USER
 DOCKER_REPO_USER_PW
 DOCKER_CONTAINER_NAME
 
+To build UAA docker image using release artifacts:
+export USE_RELEASE_ARTIFACTS=true
```

Comment:
Should read from USE_RELEASE_ARTIFACTS set in DockerizeJenkinsfile

Reply:
This comment seems like dev information for when we want to require docker image to use release artifacts. [Line 48](https://github.com/GESoftware-CF/uaa/pull/152/files/6a65e5daa72778311f706c4b7ea5a6ca7cdab047#diff-b4d7dd8622610fcebe4ad687a457cbb7R48) is the actual implementation which does read from System env set in DockerizeJenkinsfile. Do you want me to remove the comment or update the wording?

Reply:
Oops... Sorry, didn't realize it's commented out.  Never mind.  Doesn't worth changing.